test/e2e/multi-stage/registry/timeout/timeout-ref.yaml

```diff
@@ -0,0 +1,11 @@
+ref:
+  as: timeout
+  from: os
+  commands: timeout-commands.sh
+  activeDeadlineSeconds: 10
```

Comment:
All other fields use `snake_case`.

Reply:
Is consistency with them worth diverging from the Kube property we're wrapping?

Reply:
I'd say yes.

Reply:
Pivoted to snake_case with a67ec4ba6 -> 616d675db.